specification/containerservice/Fleet.Management/update/run.tsp

```diff
@@ -39,12 +39,37 @@ enum UpdateRunProvisioningState {
   ...ResourceProvisioningState,
 }
 
+scalar FleetUpdateStrategyResourceId
+  extends Azure.ResourceManager.ResourceIdentifier<[
+    {
+      type: "Microsoft.ContainerService/fleets/updateStrategies",
+    }
+  ]>;
+
 @doc("The properties of the UpdateRun.")
 model UpdateRunProperties {
   @visibility("read")
   @doc("The provisioning state of the UpdateRun resource.")
   provisioningState?: UpdateRunProvisioningState;
 
+  @doc("""
+    The resource id of the FleetUpdateStrategy resource to reference.
+
+    When creating a new run, there are three ways to define a strategy for the run:
+    1. Define a new strategy in place: Set the "strategy" field.
+    2. Use an existing strategy: Set the "updateStrategyId" field. (since 2023-08-15-preview)
+    3. Use the default strategy to update all the members one by one: Leave both "updateStrategyId" and "strategy" unset. (since 2023-08-15-preview)
+
+    Setting both "updateStrategyId" and "strategy" is invalid.
+
+    UpdateRuns created by "updateStrategyId" snapshot the referenced UpdateStrategy at the time of creation and store it in the "strategy" field. 
+    Subsequent changes to the referenced FleetUpdateStrategy resource do not propagate. 
+    UpdateRunStrategy changes can be made directly on the "strategy" field before launching the UpdateRun.
+    """)
+  @visibility("create", "read")
```

Comment:
As long as the run has not started, this can be updated no?

Reply:
updated